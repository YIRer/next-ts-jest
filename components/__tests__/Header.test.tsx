/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

describe('Header', () => {
  it('renders the h1 title', () => {
    const header = shallow(<Header />);
    expect(header.find('a'));
    expect(header).toMatchSnapshot();
  });
});
