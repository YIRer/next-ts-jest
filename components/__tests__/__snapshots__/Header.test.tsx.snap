// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders the h1 title 1`] = `
<div>
  <Link
    href="/"
  >
    <a
      style={
        Object {
          "marginRight": 15,
        }
      }
    >
      Home
    </a>
  </Link>
  <Link
    href="/about"
  >
    <a
      style={
        Object {
          "marginRight": 15,
        }
      }
    >
      About
    </a>
  </Link>
</div>
`;
